src/multibody/liegroup/liegroup-base.hpp

```diff
@@ -154,6 +154,63 @@ PINOCCHIO_LIE_GROUP_PUBLIC_INTERFACE_GENERIC(Derived,typename)
                        const Eigen::MatrixBase<JacobianOut_t> & J,
                        const AssignmentOperatorType op) const;
 
+
+    /**
+     * @brief   Transport an input matrix to the manifold defined by the dIntegrate computation.
+     *
+     * @details This input and output has to be interpreted in terms of Lie group, not vector space:
+     *          Thus, dIntegrate(q, v, J, arg) creates a manifold manifold M given by a small variation of the configuration vector or the tangent vector into the tangent space at identity.
```

Comment:
I find this documentation a bit obscure. `dIntegrate(q, v, J, arg)` does not create a manifold. It creates the tangent space (a vector space) of a manifold at a given configuration. What does this do exactly ?

Reply:
I've changed and I do agree. The previous documentation was not correct.